doc/api/http.md

```diff
@@ -79,7 +92,11 @@ http.get(options, (res) => {
 });
 ```
 
-Alternatively, you could just opt out of pooling entirely using
+You may also use an agent for an individual request. By providing
+`{agent: false}` as an option to the `http.get()` or `http.request()`
+functions, a one-time use `Agent` with default options and no connection
```

Comment:
Here's the code https://github.com/nodejs/node/blob/master/lib/_http_client.js#L63-L75. That's how I read it.